src/main/kotlin/lotto/controller/LoggoGame.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.controller
+
+import lotto.model.LottoNumber
+import lotto.model.LottoStore
+import lotto.model.LottoTicket
+import lotto.model.PurchasedLottoTickets
+import lotto.model.RandomLottoTicketStorage
+import lotto.view.InputView
+import lotto.view.ResultView
+
+object LoggoGame {
+    private const val LOTTO_PRICE: Long = 1000
```

Comment:
```suggestion
    private const val LOTTO_PRICE: Long = 1_000
```
코틀린에서는 천단위로 언더바를 표기할 수 있습니다.

Reply:
감사합니다!!
로또 상금에서는 그렇게 표현하긴 했는데 
이 부분은 `0` 단위가 많지 않아서 그대로 표기했네요
수정하도록 하겠습니다!!